model/NotifierObserver.inc.php

```diff
@@ -152,17 +143,22 @@ public static function notify($event, $type, $ids, $extraData) {
 						$message[$key] = $val;
 					}
 				}
-				self::send($message);
+				foreach (self::$messageReceivers as $receiver) {
+					$receiver(
+						$apiKey,
+						json_encode($message, JSON_UNESCAPED_SLASHES)
+					);
+				}
```

Comment:
Can we extract this out into a separate function so we can use the same thing for the call with `$topic` above without repeating the `json_encode($message, JSON_UNESCAPED_SLASHES)` stuff?